src/lib/viewers/controls/controls-bar/ControlsBar.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './ControlsBar.scss';
+
+export type Props = {
+    children: React.ReactNode;
+};
+
+export default function ControlsBar({ children, ...rest }: Props): JSX.Element {
+    return (
+        <div className="bp-ControlsBar" {...rest}>
```

Comment:
Would there be any need to provide a custom class name as a prop as each viewer may have their own styles or would that be done at a different component?

Reply:
I don't see that being necessary at the moment. It would not be difficult to extend this component later if needed.